Marlin/src/module/probe.h

```diff
@@ -256,6 +256,13 @@ class Probe {
     static bool tare();
   #endif
 
+  // Basic functions for Sensorless Homing and Probing
+  #if ENABLED(SENSORLESS_PROBING)
```

Comment:
consider also SENSORLESS_HOMING

Reply:
Yes, here if it would be convenient. Although when I created them, they were only going to be used in code areas activated by SENSORLESS_PROBING, I'm not sure if @thinkyhead has included them in code only activated by SENSORLESS_HOMING. They were not used initially in G28, for example.

Reply:
Now that I think about it. It could be the case that the end stops were not eliminated, that homing would continue to depend on the physical switches, even if the probe had been eliminated.

Reply:
> consider also SENSORLESS_HOMING

Done.